.cirrus.yml

```diff
@@ -1,9 +1,9 @@
 container:
   cpu: 1
-  memory: 1G
+  memory: 2G
```

Comment:
When testing locally I ran out of memory so this just bumps it a tiny bit.  If need be, I can change it back.